aircmd/main.py

```diff
@@ -15,8 +16,11 @@
 
 load_dotenv()
 
+# Debug mode
+AIRCMD_DEBUG: bool = bool(getenv("AIRCMD_DEBUG", False))
```

Comment:
If we put this in GlobalSettings, it should pull from environment automatically. Can we put it there instead, and use settings.AIRCMD_DEBUG? We can set the default value there as well.